scripts/proto/lbann_proto.py

```diff
@@ -410,7 +414,8 @@ def save_model(filename, first_module, data_layout, mini_batch_size, epochs,
         metric.proto(pb.model)
     for callback in callbacks:
         callback.proto(pb.model)
-    traverse_model(first_module, lambda l: l.proto(pb.model))
+    layers = traverse_layer_graph(layers)
+    pb.model.layer.extend([l.export_proto() for l in layers])
```

Comment:
If `traverse_layer_graph` becomes a generator, this can turn into `pb.model.layer.extend([l.export_proto() for l in traverse_layer_graph(layers)])`.